index.md

```diff
@@ -1,63 +1,200 @@
- - [Animations in Flutter](animations)
- - [Using custom fonts](custom-fonts)
- - [Layered design (video)](https://www.youtube.com/watch?v=dkyY9WCGMi0)
- - [Rendering pipeline (video)](https://www.youtube.com/watch?v=UUfXWzp0-DU)
- - [Dealing with box constraints in Flutter](layout)
-
-## How-Tos
-
-Learn how to accomplish specific development tasks with Flutter.
-
- - [Building Layouts in Flutter](/tutorials/layout/)
- - [Adding Assets and Images](assets-and-images)
- - [Testing Flutter Apps](testing)
- - [Debugging Flutter Apps](debugging)
- - [Developing apps in the IntelliJ IDE](intellij-ide)
- - [Formatting Flutter code](formatting)
- - [Upgrading Flutter](upgrading)
- - [Accessing Platform and Third-Party Services](platform-services)
- - [Reading and Writing Files](reading-writing-files)
- - [Handling Text Input](text-input)
-
-## Helpful Resources
-
-More resources for Flutter developers.
-
- - [Source on GitHub](https://github.com/flutter/flutter)
- - [Architecture diagram](https://docs.google.com/presentation/d/1cw7A4HbvM_Abv320rVgPVGiUP2msVs7tfGbkgdrTy0I/edit?usp=sharing)
- - [Issue tracker](https://github.com/flutter/flutter/issues)
- - [Writing useful bug reports](bug-reports)
- - [FAQ](faq)
- - [Design principles](design-principles)
-
-## Support
+*   Be highly productive
+    *   Develop for iOS and Android from a single codebase
+    *   Do more with less code, even on a single OS, with a modern, expressive 
+language and a declarative approach
+    *   Prototype and iterate easily
+        *   Experiment by changing code and reloading as your app runs (with hot reload)
+        *   Fix crashes and continue debugging from where the app left off
+*   Create beautiful, highly-customized user experiences
+    *   Benefit from a rich set of Material design widgets built using
+        Flutter's own framework
+    *   Realize custom, beautiful, brand-driven designs, without the limitations 
+of OEM widget sets
+
+## Core principles
+
+Flutter includes a functional-reactive framework, a 2D rendering engine, ready-
+made widgets, and development tools. The components work together to help you 
+design, build, test, and debug apps. Everything is organized around a few core 
+principles. 
+
+### Everything's a Widget
+
+Widgets are the basic building blocks of every Flutter app. Each widget is an 
+immutable declaration of part of a user interface.  Unlike other frameworks that 
+separate views, controllers, layouts, and other properties, Flutter has a 
+consistent, unified object model: the widget.
+
+A widget can define:
+
+*   a structural element (like a button or menu)
+*   a stylistic element (like a font or color scheme)
+*   an aspect of layout (like padding)
+*   some business logic 
+*   and so on...
+
+Widgets form a hierarchy based on composition.  Each widget nests inside, and 
+inherits properties from, its parent.  There is no separate "application" object.
+Instead, the root widget serves this role.
+
```

Comment:
True - it doesn't show b/c Markdown is smart, but removed it so no one trips over it again.